vgen/src/test/java/io/gapi/vgen/testdata/go_library.baseline

```diff
@@ -75,6 +76,22 @@ var (
 	bookPathTemplate = gax.MustCompilePathTemplate("shelves/{shelf}/books/{book}")
 )
 
+func idempotentRetryCodes() []gax.CallOption {
+	return []gax.CallOption{
+		gax.WithRetryCodes([]codes.Code{
+			codes.DeadlineExceeded,
```

Comment:
DeadlineExceeded could mean that an internal backend failed to meet its deadline, or it could mean that the whole RPC (back to the original client) timed out. I hope the retry logic handles the second case correctly.